server/src/main/java/org/opensearch/transport/RemoteClusterAwareClient.java

```diff
@@ -66,6 +72,9 @@ protected <Request extends ActionRequest, Response extends ActionResponse> void
         Request request,
         ActionListener<Response> listener
     ) {
+        final SpanScope spanScope = tracer.startSpan(SPAN_NAME_PREFIX_REMOTE_TRANSPORT_ACTION + action.name());
```

Comment:
It does not seem to be the right place, TransportService::sendRequest seems to be the one which needs instrumentation